VideoLocker/src/main/java/org/edx/mobile/base/MainApplication.java

```diff
@@ -71,11 +72,12 @@ private void init() {
         injector = RoboGuice.getOrCreateBaseApplicationInjector((Application) this, RoboGuice.DEFAULT_STAGE,
                 (Module) RoboGuice.newDefaultRoboModule(this), (Module) new EdxDefaultModule(this));
 
+        final Config config = injector.getInstance(Config.class);
 
-        Config config = injector.getInstance(Config.class);
         // initialize Fabric
         if (config.getFabricConfig().isEnabled() && !BuildConfig.DEBUG) {
             Fabric.with(this, new Crashlytics());
+            EventBus.getDefault().register(new CrashlyticsCrashReportObserver());
```

Comment:
Interesting technique to use the event bus to ensure dependency setup. Although, I guess since we don't need to listen to this event anywhere else, you could also achieve the same effect via a static utility method and static flag for enabling it.